lib/cmd/fh3/services.js

```diff
@@ -38,11 +39,36 @@ function services(argv, cb) {
   } else if ("read" === action) {
     if (args.length !== 2) return cb(services.usage);
     return readService(args[1], cb);
+  } else if ("data-sources" === action) {
+    if (args.length !== 2) return cb(services.usage);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)